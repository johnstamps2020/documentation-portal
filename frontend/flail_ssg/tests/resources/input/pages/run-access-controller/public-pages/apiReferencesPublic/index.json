{
  "$schema": "/frontend/page-schema.json",
  "title": "Automated redirect",
  "includeInBreadcrumbs": false,
  "class": "threeCards",
  "template": "redirect",
  "items": [
    {
      "label": "Banff",
      "page": "banff"
    },
    {
      "label": "Cortina",
      "page": "cortina"
    },
    {
      "label": "Dobson",
      "page": "dobson"
    },
    {
      "label": "_redirect",
      "link": "/apiReferences/dobson"
    }
  ]
}
