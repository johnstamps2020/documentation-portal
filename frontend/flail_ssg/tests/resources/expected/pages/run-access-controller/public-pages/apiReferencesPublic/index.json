{
  "$schema": "/frontend/page-schema.json",
  "title": "Automated redirect",
  "includeInBreadcrumbs": false,
  "class": "threeCards",
  "template": "redirect",
  "items": [
    {
      "label": "Banff",
      "page": "banff",
      "public": false,
      "internal": false
    },
    {
      "label": "Cortina",
      "page": "cortina",
      "public": true,
      "internal": false
    },
    {
      "label": "Dobson",
      "page": "dobson",
      "public": true,
      "internal": false
    },
    {
      "label": "_redirect",
      "link": "/apiReferences/dobson"
    }
  ],
  "public": true,
  "internal": false
}
